src/main/java/org/folio/rest/impl/BankingInformationAPI.java

```diff
@@ -0,0 +1,73 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.LogManager;
+import org.folio.rest.jaxrs.model.BankingInformation;
+import org.folio.rest.jaxrs.resource.OrganizationsBankingInformation;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+import org.apache.logging.log4j.Logger;
+import org.folio.service.organization.BankingInformationService;
+import org.folio.spring.SpringContextUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static io.vertx.core.Future.succeededFuture;
+import static org.folio.config.Constants.OKAPI_URL;
+
+public class BankingInformationAPI extends BaseApi implements OrganizationsBankingInformation {
+
+  private static final Logger logger = LogManager.getLogger(BankingInformationAPI.class);
+  private static final String BANKING_INFORMATION_LOCATION_PREFIX = "/organizations/banking-information/%s";
+
+  @Autowired
+  private BankingInformationService bankingInformationService;
+
+  public BankingInformationAPI() {
+    SpringContextUtil.autowireDependencies(this, Vertx.currentContext());
+  }
+
+  @Override
+  public void getOrganizationsBankingInformation(String query, String totalRecords, int offset, int limit, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
I guess name should be getOrganizationsBankingInformationCollection.
Can we add log debug also

Reply:
Added log